libc/test/UnitTest/CMakeLists.txt

```diff
@@ -18,6 +18,13 @@ function(add_unittest_framework_library name)
     set(library_type STATIC)
   endif()
 
+  # When the clock target is available, it will be used inside the test lib
+  # to indicate the performance of the tests.
+  set(clock_target "")
+  if(TARGET libc.src.time.clock)
```

Comment:
I forget if this is broken. I think it still makes the target even if the entrypoint is disabled so you need to check the entrypoints list.